test/kubernetes/e2e/features/services/tcproute/testdata/multi-backend-service.yaml

```diff
@@ -1,9 +1,10 @@
 apiVersion: v1
 kind: Service
 metadata:
-  name: foo
+  name: multi-svc-1
+  namespace: multi-tcp-route
```

Comment:
This is a shift, in that we are hardcoding the namespace here, instead of relying on the caller to supply the namesapce when applying the file. Can you help me understand this change?

Reply:
Resolved in commit https://github.com/solo-io/gloo/commit/281ace000966aa7f15d960ab73d60efc4fa1c189 by removing the ns definition in testdata manifests and passing it as an arg in `applyManifests()`.